src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
```

Comment:
It probably means the projectile loses all force before it reaches the target. In ranged.cpp, you've got `Creature::projectile_attack` - this deals with most of the projectile calculations.

Pretty sure just adding a "JET" flag to the ammo or adding an ammo effect "JET" to the launcher should be enough. "JET" flag has only one effect: negating the usual mechanic of projectile stopping when its damage reaches 0.